### PR TITLE
Renamed Membership to MembershipApplication

### PR DESCRIPTION
**Pull request by JeroenDeDauw**

### CODE REVIEW

File: src/Entities/MembershipApplication.php [hunk 1]
```diff
@@ -261,7 +261,8 @@ class Membership {
 	 * Set donationId
 	 *
 	 * @param integer $donationId
-	 * @return self
+	 *
+*@return MembershipApplication
```
Comment (JeroenDeDauw) [inline src/Entities/MembershipApplication.php]: #FailStorm

File: src/Entities/MembershipApplication.php [hunk 2]
```diff
@@ -588,7 +603,8 @@ public function getWikimediumShipping() {
 	 * Set membershipType
 	 *
 	 * @param string $membershipType
-	 * @return self
+	 *
+	 	 * @return self
```
Comment (manicki) [inline src/Entities/MembershipApplication.php]: wut

File: src/Entities/Subscription.php [hunk 3]
```diff
@@ -236,7 +236,7 @@ public function getCreatedAt() {
 
 	/**
 	 * @param \DateTime $createdAt
-	 * @return Membership
+	 * @return MembershipApplication
```
Comment (manicki) [inline src/Entities/Subscription.php]: `@return self`?

File: src/Entities/MembershipApplication.php (hunk 2 above)
Comment (JeroenDeDauw) [inline src/Entities/MembershipApplication.php]: Pffft, such a FailStorm! https://youtrack.jetbrains.com/issue/WI-31077#

**Pull request merged**